nifi-extension-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/GenerateFlowFile.java

```diff
@@ -187,11 +188,13 @@ protected Collection<ValidationResult> customValidate(ValidationContext validati
                     + "text and Unique FlowFiles must be false.").build());
         }
 
+        results.add(StandardValidators.DATA_SIZE_VALIDATOR.validate("File Size", validationContext.getProperty(FILE_SIZE).evaluateAttributeExpressions().getValue(), validationContext));
```

Comment:
This line should also be removed after reverting the validator change.

Reply:
Removed.